src/components/Questions/Palette/QuestionPalette.vue

```diff
@@ -80,7 +84,15 @@ export default defineComponent({
       props.hasQuizEnded ? "Skipped" : "Not Visited"
     );
 
+    const state = reactive({
+      instructionTextClass:
+        "text-lg md:text-xl lg:text-2xl mx-4 m-2 leading-tight whitespace-pre-wrap text-slate-500",
+      titleTextClass:
+        "text-lg md:text-xl lg:text-2xl mx-4 mt-10 m-2 font-bold leading-tight whitespace-pre-wrap",
+    })
```

Comment:
why reactive?

Reply:
removing it, i had copy pasted code from some other `.vue` file which had `reactive`. It serves no purpose here.